sdk/avalon_sdk/connector/direct/jrpc/jrpc_worker_registry.py

```diff
@@ -94,7 +98,16 @@ def worker_lookup(self, worker_type=None, organization_id=None,
         }
 
         if worker_type is not None:
-            json_rpc_request["params"]["workerType"] = worker_type.value
+            if isinstance(worker_type, WorkerType):
+                json_rpc_request["params"]["workerType"] = worker_type.value
+            elif isinstance(worker_type, int) and \
+                    worker_type in range(1, 3):
+                json_rpc_request["params"]["workerType"] = worker_type
+            else:
+                return create_error_response(
+                    JRPCErrorCodes.INVALID_PARAMETER_FORMAT_OR_VALUE,
+                    id,
+                    "WorkType should be an Integer")
```

Comment:
You could move below work_type validation to a function and reuse it in `worker_lookup()` and `worker_lookup_next()`

Reply:
done